events/utils.py

```diff
@@ -6,11 +6,14 @@
 from dateutil.parser import parse as dateutil_parse
 from django.conf import settings
 from django.db import transaction
+from django_orghierarchy.models import Organization
 from rest_framework.exceptions import ParseError
 
-from events.models import Keyword, Place
+from events.models import DataSource, Keyword, Place
 from events.sql import count_events_for_keywords, count_events_for_places
 
+DEFAULT_ORGANIZATION_ID = "others"
```

Comment:
I think it might make more sense to have this in `settings` rather than `utils`.